charts/reform-scan-blob-router/values.yaml

```diff
@@ -10,6 +10,8 @@ java:
         - storage-account-name
         - storage-account-primary-key
         - storage-account-secondary-key
+        - bulk-scan-storage-account-name
+        - bulk-scan-storage-account-primary-key
```

Comment:
if the keys with `cft` prefix are preferable, I'm happy to rename them.

Reply:
There is no change in `bootstrap.yaml` so not sure adding it here and we are setting that value to ? Also if we update `values.yaml` we need to bump chart version. I would say we don't need any change to this file at this point and best to update this when you use it.

Reply:
ah. yes, i refrained from chart jiggery here..